app/policies/proposal_policy.rb

```diff
@@ -98,7 +98,7 @@ def pending_delegate?
 
   def pending_approval!
     check(self.pending_approver? || self.pending_delegate?,
-          "A response has already been logged a response for this proposal")
+          "A response has already been logged for this proposal")
```

Comment:
I take your point on the fact that there are multiple approvals. I am imagining receiving this message when trying to approve an already approved proposal, and I imagine that this precise message would not tell me why my approval did not go through. Ideas on other wording? Ok to keep as-is, I just don't know if our users will know what "response" means in the context of C2